build/infrastructure/main/azfun-functionhost.tf

```
@@ -53,7 +53,14 @@ module "azfun_functionhost" {
     CREATE_LINK_REPLY_QUEUE_NAME                         = local.CREATE_LINK_REPLY_QUEUE_NAME
     CONSUMPTION_METERING_POINT_CREATED_TOPIC_NAME        = local.CONSUMPTION_METERING_POINT_CREATED_TOPIC_NAME
     CONSUMPTION_METERING_POINT_CREATED_SUBSCRIPTION_NAME = local.CONSUMPTION_METERING_POINT_CREATED_SUBSCRIPTION_NAME  
+    
+    # Message Hub
     POST_OFFICE_TOPIC_NAME                               = module.sbt_post_office.name
+    MESSAGEHUB_STORAGE_CONNECTIONSTRING                  = data.azurerm_key_vault_secret.messagehub_storage_connectionstring.value
```

Perhaps we should do this to align with the existing ones?
```suggestion
    MESSAGEHUB_STORAGE_CONNECTION_STRING                  = data.azurerm_key_vault_secret.messagehub_storage_connectionstring.value
```